lidar:
  ros__parameters:
    lidar_height: 100
    num_lasers: 32
    alpha_begin: -30.0
    alpha_end: 30.0
    laser_range: 25.0
    horizontal_step: 0.0174533
